quickwit-index-config/resources/tests/index_object/hdfs-logs.toml

```diff
@@ -0,0 +1,39 @@
+index_id = "hdfs-logs"
+index_uri = "s3://quickwit-indexes/hdfs-logs"
+
+[indexing_settings.commit_policy]
+key = "value"
+
+[indexing_settings.demux_policy]
+key = "value"
+
+[indexing_settings.merge_policy]
+key = "value"
+
+[indexing_settings.retention_policy]
+key = "value"
+
+[indexing_settings.resources]
+num_threads = 1
+heap_size = "1G"
+
+[schema] # or mapping? mappings? 
+default_search_fields = [ "body", "title" ]
+store_source = true
+demux_field = "tenant_id"
```

Comment:
Do we want this in `indexing_settings`?

Reply:
Question for @fmassot... i am not sure of the implication...